tests/integration/test_kube_ovn.py

```diff
@@ -674,3 +722,13 @@ def parse_tc_show(stdout):
     # Limit value directly follows the string limit
     limit_value = netem_split[limit_index + 1]
     return int(limit_value)
+
+
+async def curl_from_unit(ops_test, unit, ip_to_curl):
+    cmd = (
+        f" ssh --pty=false -m {ops_test.model_full_name} {unit.name} -- "
```

Comment:
somehow a space snuck in there
```suggestion
        f"ssh --pty=false -m {ops_test.model_full_name} {unit.name} -- "
```